### PR TITLE
starting with seo

### PR DESCRIPTION
**Pull request by elgatoflaco**
Added the react-helmet module for the seo meta tags. Applied in home

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,27 +8,28 @@
...
     "react-router-dom": "^4.2.2",
     "react-scripts": "^1.1.1",
     "styled-components": "^2.2.3",
     "text-gradient": "^0.2.0",
-    "typography": "^0.16.6",
-    "extract-react-intl-messages": "^0.7.1",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.3.0",
-    "prettier": "^1.7.4",
-    "react-app-rewire-styled-components": "^3.0.0",
```
Comment (Varunram) [inline package.json]: This package caused lots of issues earlier, are you sure you want to remove this?

File: package.json [hunk 2]
```diff
@@ -8,27 +8,28 @@
...
-    "react-app-rewired": "^1.3.4"
```
Comment (Varunram) [inline package.json]: Same as this one

File: package.json [hunk 3]
```diff
@@ -8,27 +8,28 @@
     "apollo-link-http": "^1.3.3",
     "d3-scale": "^2.0.0",
     "eslint-plugin-react": "^7.7.0",
+    "extract-react-intl-messages": "^0.7.1",
     "graphql": "^0.13.0",
     "graphql-tag": "^2.7.3",
     "graphql-tools": "^2.20.2",
+    "husky": "^0.14.3",
```
Comment (Varunram) [inline package.json]: husky is some pre-commit tool, right?

File: package.json [hunk 4]
```diff
@@ -8,27 +8,28 @@
     "apollo-link-http": "^1.3.3",
     "d3-scale": "^2.0.0",
     "eslint-plugin-react": "^7.7.0",
+    "extract-react-intl-messages": "^0.7.1",
```
Comment (Varunram) [inline package.json]: no idea why this is here too, required by helmet?

File: package.json (hunk 1 above)
Comment (jjenzz) [reply package.json]: He hasn’t removed these. Looks like he’s just alphabetisized the file 😊

**Pull request merged**